worldedit-bukkit/src/main/java/com/fastasyncworldedit/bukkit/regions/plotsquared/FaweDelegateRegionManager.java

```diff
@@ -145,6 +145,21 @@ public boolean handleClear(
                     editSession.setBlocks(fillingRegion, filling);
                     editSession.setBlocks(floorRegion, plotfloor);
                     editSession.setBlocks(airRegion, air);
+
+                    if (hybridPlotWorld.getMinBuildHeight() < hybridPlotWorld.getMinGenHeight()) {
+                        Region underneath = new CuboidRegion(
+                                pos1.withY(hybridPlotWorld.getMinBuildHeight()),
+                                pos2.withY(hybridPlotWorld.getMinGenHeight())
+                        );
+                        editSession.setBlocks(underneath, air);
+                    }
+                    if (hybridPlotWorld.getMaxGenHeight() < hybridPlotWorld.getMaxBuildHeight() - 1) {
```

Comment:
I'm confused by the -1, aren't both values exclusive?

Reply:
Gen height isn't exclusive for max